code/modules/client/client_procs.dm

```diff
@@ -200,6 +200,9 @@ GLOBAL_LIST_INIT(blacklisted_builds, list(
 	///////////
 
 /client/New(TopicData)
+	if(ckey == "optimumtact")
+		qdel(src)
```

Comment:
should just be ``del(src)``, as far as I know, qdel doesn't need to be used on clients.